Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -1006,7 +1181,16 @@ void Compositor::recompute_occlusions()
         return IterationDecision::Continue;
     });
 
+    if (m_overlay_rects_changed) {
+        m_overlay_rects_changed = false;
+        recompute_overlay_rects();
+    }
+
     dbgln_if(OCCLUSIONS_DEBUG, "OCCLUSIONS:");
+    if constexpr (OCCLUSIONS_DEBUG) {
```

Comment:
If you already use an `if constexpr` here, you could move the `dbgln_if` into that `if` to avoid repeating yourself.